application-forum-ui/src/main/resources/ForumCode/ConfigurationAdmin.xml

```diff
@@ -40,20 +40,22 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{include reference="Licenses.Code.VelocityMacros"/}}
-
+{{include reference='ForumCode.Macros'/}}
 {{velocity}}
```

Comment:
The empty line between the include and the Velocity macro is important. Otherwise the Velocity macro could be considered in-line. It's also safe to leave an empty line between the include macros.